server/views/pages/plan.njk

```diff
@@ -95,14 +95,16 @@
 
   <p class="govuk-body govuk-!-margin-top-5">
       {%- if data.plan.agreementDate !== null %}
-          {%- if data.isUpdatedAfterAgreement %}
+          {%- if data.plan.agreementStatus === 'COULD_NOT_ANSWER' and data.readWrite === true %}
+              <a class='govuk-!-display-none-print' href="/update-agree-plan">{{ locale.couldNotAgreePlanLink }}</a> {{ locale.couldNotAgreePlan }}
```

Comment:
I think the display-none-print here is in the wrong place. It will hide only the link, not the following text.